telemetry/definitions/commonDefinitions.json

```diff
@@ -1311,6 +1311,153 @@
+        {
+            "name": "codeTransformRunTimeLatency",
+            "type": "int",
+            "description": "A millisecond value of the total run time"
+        },
+        {
+            "name": "codeTransformStartSrcComponents",
+            "type": "string",
+            "description": "Names of components that can start a transformation",
+            "allowedValues": [
+                "devToolsStartButton",
+                "bottomPanelSideNavButton",
+                "chatPrompt"
+            ]
+        },
+        {
+            "name": "codeTransformCancelSrcComponents",
+            "type": "string",
+            "description": "Names of components that can cancel a transformation",
+            "allowedValues": [
+                "apiError",
+                "loadingPanelStopButton",
+                "devToolsStopButton",
+                "bottomPanelSideNavButton"
+            ]
+        },
+        {
+            "name": "codeTransformVCSViewerSrcComponents",
+            "type": "string",
+            "description": "Names of components that can initiate the diff viewer",
+            "allowedValues": [
+                "toastNotification",
+                "treeView",
+                "treeViewHeader"
+            ]
+        },
+        {
+            "name": "codeTransformPatchViewerCancelSrcComponents",
+            "type": "string",
+            "description": "Names of components that can cancel the diff viewer",
+            "allowedValues": [
+                "apiError",
+                "cancelButton"
+            ]
+        },
+        {
+            "name": "codeTransformApiNames",
+            "type": "string",
+            "description": "Names of allowed API calls",
+            "allowedValues": [
+                "ResumeTransformation",
+                "StartTransformation",
+                "StopTransformation",
+                "GetTransformation",
+                "GetTransformationPlan",
+                "UploadZip",
+                "CreateUploadUrl",
+                "ExportResultArchive"
+            ]
+        },
+        {
+            "name": "codeTransformPreValidationError",
+            "type": "string",
+            "description": "Names of the pre-validation errors that can occur",
+            "allowedValues": [
+                "NoPom",
+                "NoJavaProject",
+                "MixedLanguages",
+                "UnsupportedJavaVersion",
+                "ProjectJDKDiffersFromMavenJDK",
+                "NonMavenProject",
+                "EmptyProject",
+                "NonSsoLogin",
+                "RemoteRunProject"
+            ]
+        },
+        {
+            "name": "codeTransformMavenBuildCommand",
+            "type": "string",
+            "description": "Type of maven command",
+            "allowedValues": [
+                "mvnw.cmd",
+                "mvnw",
+                "mvn",
+                "IDEBundledMaven"
+            ]
+        },
+        {
```

Comment:
This is the only new update, the rest of changes must be a spacing change?

Reply:
? definitely not a spacing change. there are lots of duplicate metrics added in this PR

Reply:
I didn't catch this. My main branch fork had the duplicates on the remote. FIXED.